app-office/visidata/visidata-2.1.1.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Terminal spreadsheet multitool for discovering and arranging data"
+HOMEPAGE="http://visidata.org"
+SRC_URI="https://github.com/saulpw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/python-dateutil[${PYTHON_USEDEP}]"
+DEPEND=""
+BDEPEND="
+	test? (
+		dev-vcs/git
+		dev-python/h5py[${PYTHON_USEDEP}]
+		dev-python/openpyxl[${PYTHON_USEDEP}]
+		dev-python/pandas[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+		$(python_gen_impl_dep sqlite)
+		${RDEPEND}
+	)
+"
+
+python_test() {
+	rm tests/load-http.vd || die "Could not remove network-dependent test."
+	rm tests/graph-cursor-nosave.vd || die "Could not remove network-dependent test."
+	rm tests/messenger-nosave.vd || die "Could not remove network-dependent test."
+	rm tests/save-benchmarks.vd || die "Could not benchmarks test"
+	rm tests/graph-sincos-nosave.vd || die "Could not benchmarks test"
+	rm tests/graphpr-nosave.vd || die "Could not benchmarks test"
+	rm tests/describe-error.vd || die "Could not remove network-dependent test"
+	rm tests/describe.vd || die "Could not remove network-dependent test"
+	rm tests/edit-type.vd || die "Could not remove network-dependent test"
+	git init || die "Git init failed."
+	git add tests/golden/ || die "Git add failed."
+	dev/test.sh || die "Tests failed."
+	rm .git -rf || die "Could not clean up git test directory."
+}
+
+pkg_postinst() {
+	optfeature "integration with yaml" >=dev-python/pyyaml-5.1
```

Comment:
To use `optfeature` you should inherit the `optfeature` eclass (It is no longer included in `eutils`)